### ISSUE TITLE
TextArea has both placeholder and aria-placeholder, violating accessibility rules

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon#7234]
Go here: https://react.carbondesignsystem.com/?path=/story/textarea--default

the textarea component has both placeholder and aria-placeholder attributes, which apparently it's an accessibility bug.

![image](https://user-images.githubusercontent.com/10033254/98297195-a7b39480-1f79-11eb-8ed8-2987570052cc.png)

**Comment by tw15egan** [comment]
Should be fixed via https://github.com/carbon-design-system/carbon/pull/7156 in our next release, if not, feel free to reopen. Thanks for reporting!
